mat/format.go

```diff
@@ -274,21 +274,21 @@ func formatMATLAB(m formattable, prefix string, _ int, _ byte, squeeze bool, fs
 		return
 	}
 
+	var buf = make([]byte, 0, 4)
```

Comment:
I don't think this presizing to 4 makes sense here since in the default case `prefix` is arbitrarily long with a minumum of two. The remainder are one byte long, so maybe just presize to 2, `buf = make([]byte, 0, 2)` and let it grow if needed in the default case when `len(prefix) != 0`.